saas_portal/controllers/main.py

```diff
@@ -30,6 +30,7 @@ def book_then_signup(self, **post):
         dbtemplate = self.get_template()
         client_id = self.get_new_client_id(full_dbname)
         request.registry['oauth.application'].create(request.cr, SUPERUSER_ID, {'client_id': client_id, 'name':full_dbname})
+        # FIXME: line below should be deleted. This route called book_then_signup, but work as if user already signed up
```

Comment:
Could you work on this. I think some refactoring is needed. 
page/start doesn't work now due to this reason